_posts/2018-02-26-success-with-serverless.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+status: publish
+published: true
+title: 'Success With Serverless'
+author:
+  display_name: Scott Davis
+  email: stdavis@utah.gov
+date: 2018-02-26 23:45:28
+categories:
+- Developer
+tags:
+- nodejs
+---
+
+In this post, I'd like to describe a recent success story that we've had using [serverless computing](https://en.wikipedia.org/wiki/Serverless_computing). This new(ish) technology has some exciting advantages over traditional solutions. We were looking for an excuse to give it a try and found a great fit.
+
+### The Problem
+[AGRC's base maps]({{ "/data/base-map-and-imagery/" | prepend: site.baseurl }}) (including the [Google imagery]({{ "/discover/license/" | prepend: site.baseurl }})) are served via a custom server application called Giza. Part of the advantage of this solution is the ability to secure and track usage via quad-words. These unique words are assigned to a specific user and are locked down to a specific domain or IP address. For example, if my quad-word is locked down to <code>mydomain.com</code>, then requests originating from any other domain or IP address are blocked by the server. This prevents unauthorized access of licensed content as well as allows AGRC to track analytics.
```

Comment:
_"AGRC’s base maps (including the Google imagery)"_:  Can we remove the "the" here? 
_"Part of the advantage of this solution"_: I'm not sure what "solution" is referring to here. Do we mean serverless computing or Giza?

Reply:
We usually use the "the" when referring to the Google imagery. It's a specific set of data that the state has purchased. Maybe we are incorrect in doing so?

Solution is referring to Giza.

Reply:
Oh, got it! That makes sense about "the Google imagery". K, I think it's fine if you leave it as it is then. And for the solution wording, maybe revise (for clarity) to: "Part of the advantage of using Giza is that it allows you to secure and track usage via quad-words." Would that work?